apps/pay/components/Currency/currency-dropdown.tsx

```diff
@@ -1,6 +1,7 @@
+"use client"
```

Comment:
maybe in a follow up PR, let's also make the filename lower case only, so that it's consistent with the rest of the codebase

Reply:
Yup, agree.